plugins/templates/labhub/errors/not-eligible-invite.jinja2.md

```diff
@@ -0,0 +1,2 @@
+- A newcomer cannot invite other members to the {{ organization }} org.
+- You need to be a maintainer in order to invite someone to team developers and maintainers.
```

Comment:
Move this line into a new template and return this when needed as mentioned [here](https://github.com/coala/corobo/pull/537/files#r192590984).

Reply:
`team developers and maintainers` --> `developers or maintainers team`